### ISSUE TITLE
More sources!

### ISSUE THREAD

**Issue by arquinn** [paivaspol/EECS583#5]
Merged vaspol's branch into master; Added the valgrind output for all of the spec benchmarks, except for one that was taking about 40 minutes for valgrind to run. The outputs are all in the same place:

"valgrind_output"/
   [benchmark].data.results
   [benchmark].instructions.results
   [benchmark].not.found

Vaspol, there appears to be a bunch of stuff in the not.found files, I'm going to parse it all out and make sure it makes sense, but take a look whenever you get a chance.

**Comment by paivaspol** [comment]
@arquinn Cool, thanks. I'll look at those functions and extract them.

**Comment by paivaspol** [comment]
Quick question. The ones that are marked as "I already found this!" means that I can ignore those functions right?

**Comment by arquinn** [comment]
Yeah, there shouldn't be any of those in there though. I'll take a look.
